### DISCUSSION TITLE
Testing - reverting database to state before tests - admin user password

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#11628] [Q&A] [answered]
When we run codeception unit tests, by default database is set to an empty state when tests finish, with just one user.

What is the password of this user?

**Comment by angrybrad** [comment, marked as answer]
https://github.com/craftcms/cms/blob/develop/src/test/TestSetup.php#L489-L495

user: craftcms
pass: `craftcms2018!!`
